packages/website/src/remark/tabs.js

```diff
@@ -0,0 +1,165 @@
+/*
+ * original code https://github.com/mccleanp/remark-docusaurus-tabs
+ *
+ * this code has been updated to properly escape values
+ *
+ * MIT License
+ *
+ * Copyright (c) 2020 Paul McClean
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+function tabs() {
+  function renderTabs(tabs, nodes) {
+    function getId(tab) {
+      return nodes[tab.start].data.id;
+    }
+
+    function getLabel(root) {
+      return root.children
+        .map(item => {
+          switch (item.type) {
+            case 'text':
+            case 'inlineCode':
+              return item.value;
+            case 'strong':
+              return getLabel(item);
+            default:
+              console.error(item);
+              throw new Error(`[TABS] unsupported type ${item.type}`);
+          }
+        })
+        .join('')
+        .trim();
+    }
+
+    let tabNodes = [];
+
+    tabNodes.push({
+      type: 'jsx',
+      value: `<Tabs defaultValue="${getId(tabs[0])}" values={[${tabs
```

Comment:
In recent Docusaurus versions you don't need to use `defaultValue` and `values` in `Tabs` anymore (https://docusaurus.io/docs/markdown-features/tabs), so `defaultValue` will be inferred as the first tab, and you could just move the `label` prop to each `TabItem` node.